src/rpc_client/src/meta_client.rs

```diff
@@ -146,7 +146,7 @@ impl MetaClient {
         let resp = self.inner.heartbeat(request).await?;
         if let Some(status) = resp.status {
             if status.code() == risingwave_pb::common::status::Code::UnknownWorker {
-                panic!("{}", status.message);
+                panic!("worker expired: {}", status.message);
```

Comment:
It seems that sometimes the panic message not printed to console, please add `error!` here. Also panic! doesn't guarantee to stop the process, maybe we should use `exit`.